compiler/runtime/runtime64_specific.cpp

```diff
@@ -1,3 +1,7 @@
+#ifndef SKIP64
+#define SKIP64
+#endif
```

Comment:
As mentioned on the Makefile, it seems odd to do this here for cpp files and using the build system for c files.

Reply:
Ok, I'll do something in this PR then